opencti-platform/opencti-front/src/private/components/common/form/AuthorizedMembersField.tsx

```diff
@@ -70,15 +70,16 @@ const AuthorizedMembersField = ({
   form,
   field,
   owner,
-  showAllMembersLine = false,
   showCreatorLine = false,
   canDeactivate = false,
 }: AuthorizedMembersFieldProps) => {
   const { t_i18n } = useFormatter();
-  const { me } = useAuth();
+  const { me, settings } = useAuth();
   const { setFieldValue } = form;
   const { name, value } = field;
 
+  const showAllMembersLine = !settings.platform_organization?.id;
```

Comment:
Just to be sure: we don't want to show "All" when a platform orga is set, right ?

Reply:
Exactly :) BUT I think I missed a part, where this is only valid for Knowledge and not Dashboards for example

Reply:
Yes, ALL is not visible anymore on custom dashboards when a platform orga is set.